client/lib/plans/index.js

```diff
@@ -310,6 +311,25 @@ export function planMatches( planKey, query = {} ) {
 	return match( 'type' ) && match( 'group' ) && match( 'term' );
 }
 
+export function calculateMonthlyPriceForPlan( planSlug, termPrice ) {
+	return calculateMonthlyPrice( getPlan( planSlug ).term, termPrice );
+}
+
+export function calculateMonthlyPrice( term, termPrice ) {
+	let divisor;
+	if ( term === TERM_MONTHLY ) {
```

Comment:
might consider a switch here instead of chained ifs. bit easier to read?

Reply:
It started as a switch and then I refactored to this because of all the extra lines for `break;` :-)

Reply:
if you're shooting for concise, a lookup map would be really small.

```
const TERM_TO_DIVISOR = {
  [ TERM_MONTHLY ]: 1
  [ TERM_ANNUALLY ]: 12,
  [ TERM_BIENNIALLY ]: 24
};

const divisor = TERM_TO_DIVISOR[ term ];
if ( ! divisor ) { throw new ... }
```

Reply:
but not essential here. just taste.

Reply:
or just inline those `break`'s with a `return`…

```js
const term2months = term => {
	switch ( term ) {
		case TERM_MONTLY: return 1;
		case TERM_ANNUALLY: return 12;
		case TERM_BIENNIALLY: return 24;
		default:
			throw …
	}
}
```

can't beat a switch for performance (won't matter here but it _is_ less pressure on the garbage collector than allocating new objects) and if we all used them then we could have a nice consistent way of dealing with these kind of things.